src/blocks/small-search.css

```diff
@@ -0,0 +1,45 @@
+.small-search__input {
+  font-weight: 400;
+  font-size: 14px;
+  margin: 0 auto;
+  width: 100%;
+  text-indent: 33px;
+  border: none;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11;
+  background-position: 13 14;
+}
+
+.main__small-search {
+  display: flex;
+  box-sizing: border-box;
+  margin: 20px 8px 0;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.small-search__input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11;
```

Comment:
The 'background-size' property is missing a unit. It should be specified in a valid CSS unit like 'px', 'em', etc.